shared/scss/components/cms/items.scss

```diff
@@ -27,6 +27,7 @@
   margin: 0;
   font-weight: 600;
   font-family: $font-heading;
+  font-size: $font-size-large;
```

Comment:
Hmm, I was quite unsure whether this is the right place for defining the font size, especially as I had to make the same change in another ticket https://github.com/6aika/sixodp/pull/348/files ... Should it rather have been changed in the headings.scss where the font-size is defined for other heading levels (I don't know if there are some h2 headings that we don't want to make smaller):
`h3, h4, h5, h6, .module-heading {
  @include heading;
  font-size: $font-size-large;

  i {
    display: none;
  }
}
`

Reply:
This is probably fine, as we don't want to change all of them, rather we want to to keep the same look for it.